modules/shapes/widgets/word-art.php

```diff
@@ -0,0 +1,690 @@
+		$this->start_controls_section( 'section_content_wordart', [
+			'label' => __( 'WordArt', 'elementor' ),
+			'tab' => Controls_Manager::TAB_CONTENT,
+		] );
+
+		$this->add_control( 'text',
+			[
+				'label' => __( 'Text', 'elementor' ),
+				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
+				'default' => __( 'Add Your Text Here', 'elementor' ),
+			]
+		);
+
+		$this->add_control( 'path', [
+			'label' => __( 'Shape Path', 'elementor' ),
+			'type' => Controls_Manager::SELECT,
+			'options' => Shapes_Module::get_paths(),
+			'default' => 'circle',
+		] );
+
+		$this->add_control( 'custom_path', [
+			'label' => __( 'SVG', 'elementor' ),
+			'type' => Controls_Manager::MEDIA,
+			'condition' => [
+				'path' => 'custom',
+			],
+			'dynamic' => [
+				'active' => true,
+			],
+			'description' => __( 'Add data-path-anchor attribute to your path in order to put the text along it. Otherwise, it will default to the first path element.', 'elementor' ),
+		] );
+
+		$this->add_control( 'shape_notice',
+			[
+				'type' => Controls_Manager::RAW_HTML,
+				'raw' => __( 'Need More Shapes?', 'elementor' ) . '<br>' . sprintf( __( 'Explore additional Premium Shape packs and use them in your site. <a target="_blank" href="%s">Learn More</a>', 'elementor' ), 'https://go.elementor.com/wordart-more-paths' ),
+				'content_classes' => 'elementor-panel-alert elementor-panel-alert-info',
+			]
+		);
+
+		$this->add_control( 'link',
+			[
+				'label' => __( 'Link', 'elementor' ),
+				'type' => Controls_Manager::URL,
+				'label_block' => true,
+				'dynamic' => [
+					'active' => true,
+				],
+				'placeholder' => __( 'Paste URL or type', 'elementor' ),
+			]
+		);
+
+		$this->add_responsive_control( 'alignment',
+			[
+				'label' => __( 'Alignment', 'elementor' ),
+				'type' => Controls_Manager::CHOOSE,
+				'default' => 'left',
+				'options' => [
+					'start' => [
+						'title' => __( 'Start', 'elementor' ),
+						'icon' => 'eicon-text-align-left',
+					],
+					'middle' => [
+						'title' => __( 'Middle', 'elementor' ),
+						'icon' => 'eicon-text-align-center',
+					],
+					'end' => [
+						'title' => __( 'End', 'elementor' ),
+						'icon' => 'eicon-text-align-right',
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}}' => '--alignment: {{VALUE}}',
+				],
+				'frontend_available' => true,
+			]
+		);
+
+		$this->add_control( 'show_path', [
+			'label' => __( 'Show Path', 'elementor' ),
+			'type' => Controls_Manager::SWITCHER,
+			'label_on' => __( 'On', 'elementor' ),
+			'label_off' => __( 'Off', 'elementor' ),
+			'return_value' => self::DEFAULT_PATH_FILL,
+			'separator' => 'before',
+			'default' => '',
+			'selectors' => [
+				'{{WRAPPER}}' => '--path-stroke: {{VALUE}}; --path-fill: transparent;',
+			],
+		] );
+
+		$this->add_control( 'path_notice',
+			[
+				'type' => Controls_Manager::RAW_HTML,
+				'raw' => __( 'Notice! The path will be visible on live preview', 'elementor' ),
```

Comment:
'live preview' or 'live site'?

Reply:
Waiting for the Product response.

Reply:
@shilo-ey ?

Reply:
It's an outdated code which has been removed already.